{"scope":"source.elixir","completions":[{"trigger":"BadArityError.message(${1:exception})","contents":"message(${1:exception})"}]}